README.md

```diff
@@ -115,6 +115,10 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 
 - [@vitejs/plugin-react-refresh](https://github.com/vitejs/vite/tree/main/packages/plugin-react-refresh) - Official React Refresh support.
 
+### Transformers
```

Comment:
```suggestion
### Loaders
```

I think it's better to name it "loaders" here.

Can you also add a few instructions or examples of how to use the plugin?

Thanks.